addons/uis/apps/ios.md

```diff
@@ -5,46 +5,47 @@ title: iOS openHAB App
 
 {% include base.html %}
 
-## iOS openHAB App
```

Comment:
I wasn't sure which convention to use for headings - I used ## for the main header and ### for the sub headings to be in line with what I saw on the BasicUI page, but in looking around there's some variation - I imagine you know best though! 😀

Reply:
In general articles should start with H1. The main articles should do so. I'll look into Basic UI.